templates/history-downloads.php

```diff
@@ -116,14 +121,20 @@
 		?>
 	</table>
 	<?php
-		echo edd_pagination(
-			array(
-				'type'  => 'download_history',
-				'total' => ceil( edd_count_purchases_of_customer() / 20 ) // 20 items per page
-			)
-		);
+	$count = edd_count_orders(
+		array(
+			'user_id' => get_current_user_id(),
```

Comment:
This should use the customer ID instead of the current user ID. Otherwise we query the sales above using the customer ID, but base pagination off the user ID. That could potentially lead to unexpected results.